config/routes.rb

```diff
@@ -3,7 +3,12 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create ] do
+      resources :comments, only: [:new, :create]
+    end
   end
+
+  get 'new', to: 'posts#new'
```

Comment:
**_[OPTIONAL]_** 

Rails have a design pattern that says `convention over configuration` which means everything is ready for you to use for example
```suggestion
  get '/posts/new`
```
Rails will understand that route will go to `posts` controller action `new`

- [OPTIONAL] Let's follow rails conventions